core/domain/exp_services.py

```diff
@@ -1117,9 +1114,9 @@ def update_exploration(
         committer_id: str. The id of the user who is performing the update
             action.
         exploration_id: str. The id of the exploration to be updated.
-        change_list: list(dict). A change list to be applied to the given
-            exploration. Each entry in change_list is a dict that represents an
-            ExplorationChange.
+        change_list: list(ExplorationChange). A change list to be applied to the
+            given exploration. Each entry in change_list is an ExplorationChange
```

Comment:
Drop the second sentence; it's redundant with the typeinfo.

Reply:
Done.